src/components/createDraftDialog/CreateDraftDialog.tsx

```diff
@@ -83,7 +83,7 @@ const CreatDraftDialog = ({ open, onClose, onSubmit, cqlLibrary }) => {
     } as CqlLibrary,
     validationSchema: Yup.object().shape({
       cqlLibraryName: Yup.string()
-        .max(255, "Library name cannot be more than 255 characters.")
+        .max(255, "Library name cannot be more than 64 characters.")
```

Comment:
max still set to 255

Reply:
It's on my local as 64 and it's saying it's up to date when I try to pull/push. Weird. Edited in github